### ISSUE TITLE
Add Accessibility Role to components

### ISSUE THREAD

**Issue by ztefanie** [digitalfabrik/integreat-app#2553]
**Is your feature request related to a problem? Please describe.**
For better accessibility the `role` should be set.

**Describe the solution you'd like**
Find all relevant ui elements, where the `role` should be set, and set it to the correct value. See https://reactnative.dev/docs/accessibility#role

Replace all usages of `accessibilityRole` with `role`.

**Describe alternatives you've considered**
Leave it as it is.

**Comment by sarahsporck** [comment]
Related Issue:
- #2176

**Comment by steffenkleinle** [comment]
I'd postpone this until after the react-native upgrade as it seems like there is a change from `accessibilityRole` to `role`,
